src/pages/IdlePage.tsx

```diff
@@ -37,7 +38,7 @@ const IdlePage = () => {
   }, [countdown, markVoterCardVoided, resetBallot])
 
   useInterval(() => {
-    setCountdown(t => (t > 0 ? t - 1 : 0))
+    setCountdown(countdown => countdown - 1)
```

Comment:
For this use case I don't think this matters much, but there's no guarantee that `setInterval` and `setTimeout` will actually run "on time". It's better to have an interval to refresh the display, but to make the data be based on an absolute timestamp, i.e. we calculate the idle timeout time upfront and update the display based on that.